File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/ItemBasket/Basket.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemBasket;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Basket {

    @SerializedName("items")
    @Expose
    private List<Item> items = null;
    @SerializedName("total")
    @Expose
    private int total;
    @SerializedName("weight")
    @Expose
    private String weight;
    @SerializedName("info")
    @Expose
    private String info;
    @SerializedName("goods_count")
    @Expose
    private int goodsCount;
    @SerializedName("items_count")
    @Expose
    private int itemsCount;
    @SerializedName("discount")
    @Expose
    private int discount;

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

    public int getTotal() {
        return total;
    }

    public void setTotal(int total) {
        this.total = total;
    }

    public String getWeight() {
        return weight;
    }

    public void setWeight(String weight) {
        this.weight = weight;
    }

    public String getInfo() {
        return info;
    }

    public void setInfo(String info) {
        this.info = info;
    }

    public int getGoodsCount() {
        return goodsCount;
    }

    public void setGoodsCount(int goodsCount) {
        this.goodsCount = goodsCount;
    }

    public int getItemsCount() {
        return itemsCount;
    }

    public void setItemsCount(int itemsCount) {
        this.itemsCount = itemsCount;
    }

    public int getDiscount() {
        return discount;
    }

    public void setDiscount(int discount) {
        this.discount = discount;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itembasket/Basket.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Basket {

    @SerializedName("items")
    @Expose
    var items: List<Item>? = null
    @SerializedName("total")
    @Expose
    var total: String? = null
    @SerializedName("weight")
    @Expose
    var weight: String? = null
    @SerializedName("info")
    @Expose
    var info: String? = null
    @SerializedName("goods_count")
    @Expose
    var goodsCount: Int = 0
    @SerializedName("items_count")
    @Expose
    var itemsCount: Int = 0
    @SerializedName("discount")
    @Expose
    var discount: Int = 0

}
